docs/version0.22.md

```diff
@@ -44,6 +45,9 @@ The latest builds of OpenJDK with OpenJ9 for Java 8 and 11 at the AdoptOpenJDK c
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### New `-XX:[+|-]PortableSharedCache` option added
+
+This option enables AOT compiled code to be generated based on a chosen set of processor features that maximizes its portability across machines. It is currently supported only on x86. The feature is turned on by default in Docker containers that can be disabled with `-XX:-PortableSharedCache`. For more information, see [`-XX:[+|-]PortableSharedCache`](xxportablesharedcache.md).
```

Comment:
> `that can be disabled with -XX:-PortableSharedCache`  

I would replace `that` with `but` or maybe with `and`.

Reply:
Peter is out until Wednesday. Please replace "that" with "and".